src/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,119 @@
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  CardMedia,
+  Collapse,
+  List,
+  ListItem,
+  ListItemText,
+  Typography,
+} from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import React, { useState } from 'react';
+import ReactPlayer from 'react-player';
+import { NavLink } from 'react-router-dom';
+import { save } from '../../services/localStorage/storage.js';
+import { ExpandMore } from './CourseCard.styled';
+
+const CourseCard = ({ data }) => {
+  const [hovered, setHovered] = useState(false);
+  const [expanded, setExpanded] = useState(false);
```

Comment:
```suggestion
  const [isHovered, setIsHovered] = useState(false);
  const [isExpanded, setIsExpanded] = useState(false);
```
Такі назви можуть краще дати зрозуміти, що це саме булеві значення. 
Проте в конкретному випадку це і так досить зрозуміло
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Reply:
Тут согласен если значение буливое то переменную лучше начинать с is